workbench-core/swb-common-ui/src/api/users.ts

```diff
@@ -16,6 +16,7 @@ const createUser = async (createUserForm: CreateUserForm): Promise<void> => {
   await httpApiPost(`users`, { ...createUserForm });
 };
 
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
can we explicitly define the return type here?

Reply:
The type `any` here would be correct (same reason as environments above)